app-utils/qprint/autobuild/prepare

```diff
@@ -0,0 +1,3 @@
+# Arch Linux: directories are not created during `make install`.
+mkdir -p "$PKGDIR"/usr/bin
```

Comment:
Missing verbosity and abinfo. Consider merging the commands or patching Makefile (submitting upstream).